client/components/tinymce/index.jsx

```diff
@@ -307,7 +307,7 @@ module.exports = React.createClass( {
 			toolbar1: config.isEnabled( 'post-editor/insert-menu' )
 				? 'wpcom_insert_menu,formatselect,bold,italic,bullist,numlist,link,blockquote,alignleft,aligncenter,alignright,spellchecker,wp_more,wpcom_advanced'
 				: 'wpcom_add_media,formatselect,bold,italic,bullist,numlist,link,blockquote,alignleft,aligncenter,alignright,spellchecker,wp_more,wpcom_add_contact_form,wpcom_advanced',
-			toolbar2: 'strikethrough,underline,hr,alignjustify,forecolor,pastetext,removeformat,wp_charmap,outdent,indent,undo,redo,wp_help',
+			toolbar2: 'strikethrough,hr,forecolor,pastetext,removeformat,wp_charmap,outdent,indent,undo,redo,wp_help',
```

Comment:
@timmyc do we want to incorporate these changes? If so, this PR should be good after it's rebased.